<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>ConfigDialog</class>
  <widget class="QDialog" name="ConfigDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>串口配置</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
        <widget class="QLabel" name="label">
          <property name="text">
            <string>串口:</string>
          </property>
        </widget>
      </item>
      <item row="0" column="1">
        <widget class="QComboBox" name="portComboBox"/>
      </item>
      <item row="0" column="2">
        <widget class="QPushButton" name="refreshButton">
          <property name="text">
            <string>刷新</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QLabel" name="label_2">
          <property name="text">
            <string>波特率:</string>
          </property>
        </widget>
      </item>
      <item row="1" column="1">
        <widget class="QComboBox" name="baudRateComboBox"/>
      </item>
      <item row="2" column="0">
        <widget class="QLabel" name="label_3">
          <property name="text">
            <string>数据位:</string>
          </property>
        </widget>
      </item>
      <item row="2" column="1">
        <widget class="QComboBox" name="dataBitsComboBox">
          <item>
            <property name="text">
              <string>5</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>6</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>7</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>8</string>
            </property>
          </item>
        </widget>
      </item>
      <item row="3" column="0">
        <widget class="QLabel" name="label_4">
          <property name="text">
            <string>停止位:</string>
          </property>
        </widget>
      </item>
      <item row="3" column="1">
        <widget class="QComboBox" name="stopBitsComboBox">
          <item>
            <property name="text">
              <string>1</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>1.5</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>2</string>
            </property>
          </item>
        </widget>
      </item>
      <item row="4" column="0">
        <widget class="QLabel" name="label_5">
          <property name="text">
            <string>校验位:</string>
          </property>
        </widget>
      </item>
      <item row="4" column="1">
        <widget class="QComboBox" name="parityComboBox">
          <item>
            <property name="text">
              <string>None</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Even</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Odd</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Space</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Mark</string>
            </property>
          </item>
        </widget>
      </item>
      <item row="5" column="0">
        <widget class="QLabel" name="label_6">
          <property name="text">
            <string>流控制:</string>
          </property>
        </widget>
      </item>
      <item row="5" column="1">
        <widget class="QComboBox" name="flowControlComboBox">
          <item>
            <property name="text">
              <string>None</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>RTS/CTS</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>XON/XOFF</string>
            </property>
          </item>
        </widget>
      </item>
      <item row="6" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <spacer name="horizontalSpacer">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okButton">
              <property name="text">
                <string>确定</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="text">
                <string>取消</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="horizontalSpacer_2">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections>
    <connection>
      <sender>okButton</sender>
      <signal>clicked()</signal>
      <receiver>ConfigDialog</receiver>
      <slot>accept()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>177</x>
          <y>250</y>
        </hint>
        <hint type="destinationlabel">
          <x>203</x>
          <y>179</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>cancelButton</sender>
      <signal>clicked()</signal>
      <receiver>ConfigDialog</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>296</x>
          <y>250</y>
        </hint>
        <hint type="destinationlabel">
          <x>203</x>
          <y>179</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
